cli/cmd/install.go

```diff
@@ -81,6 +118,7 @@ spec:
       annotations:
         conduit.io/created-by: "{{.CliVersion}}"
     spec:
+      serviceAccount: conduit
```

Comment:
How would you feel about using this service account for the web and prometheus pod specs as well? I don't feel strongly either way, just curious.

Reply:
We should use a different service account for each one and give each service account the minimum roles needed for it to do its job (principle of least privilege).

In particular, we shouldn't have any service account named `conduit` but instead we should have ones with names that correspond to the names of the pods they are used for.